### PR TITLE
RMIO-66 Upgrade to build script v2.0.0-alpha022

### PR DESCRIPTION
**Pull request by dukorama**

### CODE REVIEW

File: Build/Remotion.Server.build [hunk 1]
```diff
@@ -50,17 +50,21 @@
   <Target Name="TestBuild_Configuration">
     <PropertyGroup>
       <Platforms>x86</Platforms>
-      <DatabaseSystems>SqlServer2008R2</DatabaseSystems>
-      <Browsers>Chrome</Browsers>
+      <DatabaseSystems>NoDB</DatabaseSystems>
+      <Browsers>NoBrowser</Browsers>
+      <ExecutionRuntimes>Win_NET48+EnforcedLocalMachine</ExecutionRuntimes>
```
Comment (Stefan-Ilic) [inline Build/Remotion.Server.build:55]:
```suggestion
      <ExecutionRuntimes>Win_NET48</ExecutionRuntimes>
```

File: Build/Remotion.Server.build [hunk 2]
```diff
@@ -50,17 +50,21 @@
...
   </Target>
   
   <Target Name="FullBuild_Configuration">
     <PropertyGroup>
       <Platforms>x86+x64</Platforms>
-      <DatabaseSystems>SqlServer2005+SqlServer2008+SqlServer2008R2+SqlServer2012+SqlServer2014</DatabaseSystems>
-      <Browsers>Chrome+InternetExplorer</Browsers>
+      <DatabaseSystems>NoDB</DatabaseSystems>
+      <Browsers>NoBrowser</Browsers>
+      <ExecutionRuntimes>Win_NET472+Win_NET48+EnforcedLocalMachine</ExecutionRuntimes>
```
Comment (Stefan-Ilic) [inline Build/Remotion.Server.build:66]:
```suggestion
      <ExecutionRuntimes>Win_NET472+Win_NET48</ExecutionRuntimes>
```

Comment (MichaelKetting) [reply Build/Remotion.Server.build:66]: Fun fact: the various remotion projects all still support .net 4.6 (outside of trunk). so, to have complete coverage, net462 should be added to the matrix. I think I kind of forgot about that one :o

File: Build/Customizations/Projects.props [hunk 3]
```diff
@@ -1,16 +1,38 @@
...
+
+      <!-- Local-->
+      LocalMachine + NET48 + NoBrowser + NoDB + Debug + x86;
+      LocalMachine + NET48 + NoBrowser + NoDB + Release + x86;
+      LocalMachine + NET48 + NoBrowser + NoDB + Debug + x64;
+      LocalMachine + NET48 + NoBrowser + NoDB + Release + x64;
+
+      <!-- Exercise compatibility between installed .NET version, target framework and SQL Server -->
+      Win_NET48 + NET472 + NoBrowser + NoDB + Release + x64;
+      Win_NET472 + NET472 + NoBrowser + NoDB + Release + x64;
```
Comment (Stefan-Ilic) [inline Build/Customizations/Projects.props:18]: Need a NET462 test config here too

File: Build/Remotion.Server.build [hunk 4]
```diff
@@ -50,17 +50,21 @@
...
   
   <Target Name="FullBuild_Configuration">
     <PropertyGroup>
       <Platforms>x86+x64</Platforms>
-      <DatabaseSystems>SqlServer2005+SqlServer2008+SqlServer2008R2+SqlServer2012+SqlServer2014</DatabaseSystems>
-      <Browsers>Chrome+InternetExplorer</Browsers>
+      <DatabaseSystems>NoDB</DatabaseSystems>
+      <Browsers>NoBrowser</Browsers>
+      <ExecutionRuntimes>Win_NET472+Win_NET48</ExecutionRuntimes>
+      <TargetRuntimes>NET472+NET48</TargetRuntimes>
```
Comment (Stefan-Ilic) [inline Build/Remotion.Server.build:67]: Need NET462

File: Build/Remotion.build [hunk 5]
```diff
@@ -23,15 +23,28 @@
...
+  <ItemGroup>
+      <NUnitExtensionsPaths Include="$(PackagesDirectory)NUnit.Extension.NUnitV2Driver.3.8.0\"/>
+  </ItemGroup>
+
+  <ItemGroup>
+      <SupportedBrowsers Remove="@(SupportedBrowsers)"/>
+      <SupportedPlatforms Include="x86;x64"/>
+      <SupportedDatabaseSystems Remove="@(SupportedDatabaseSystems)"/>
+      <SupportedExecutionRuntimes Include="Win_NET472=$(DockerImageDotNetFrameworkRuntime4_7);Win_NET48=$(DockerImageDotNetFrameworkRuntime4_8)"/>
+      <SupportedTargetRuntimes Include="NET48;NET472;"/>
```
Comment (Stefan-Ilic) [inline Build/Remotion.build:44]: Need NET462

File: Build/Customizations/Projects.props (hunk 3 above)
Comment (MichaelKetting) [reply Build/Customizations/Projects.props:18]: Win_NET48 + NET472 + NoBrowser + NoDB + Release + x64;
  Win_NET48 + NET462 + NoBrowser + NoDB + Release + x64;
  Win_NET472 + NET472 + NoBrowser + NoDB + Release + x64;
  Win_NET472 + NET462 + NoBrowser + NoDB + Release + x64;
  Win_NET462 + NET462 + NoBrowser + NoDB + Release + x64;

File: Build/Build.csproj [hunk 6]
```diff
@@ -52,8 +52,8 @@
     <Folder Include="Properties\" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Remotion.BuildScript, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Remotion.BuildScript.2.0.0-alpha018\lib\net45\Remotion.BuildScript.dll</HintPath>
```
Comment (Stefan-Ilic) [inline Build/Build.csproj:56]: Marking it here as well, since MK commented on this

File: Build/packages.config [hunk 7]
```diff
@@ -1,6 +1,14 @@
...
+  <package id="DependDB.BuildProcessor" version="2.0.0" targetFramework="net45" />
+  <package id="DependDB.BuildProcessor.NuGetPreProcessor" version="2.0.0" targetFramework="net45" />
+  <package id="JetBrains.Annotations" version="2019.1.3" targetFramework="net45" />
+  <package id="MSBuild.Extension.Pack" version="1.5.0" targetFramework="net45" />
+  <package id="NUnit.ConsoleRunner" version="3.11.1" targetFramework="net45" />
+  <package id="NUnit.Extension.NUnitV2Driver" version="3.8.0" targetFramework="net45" />
+  <package id="NuGet.CommandLine" version="3.5.0" targetFramework="net45" developmentDependency="true" />
   <package id="NuGet.for.MSBuild" version="1.4.3" targetFramework="net45" developmentDependency="true" />
-  <package id="Remotion.BuildScript" version="2.0.0-alpha018" targetFramework="net45" developmentDependency="true" />
+  <package id="Remotion.BuildTools.MSBuildTasks" version="1.0.6862.28795" targetFramework="net45" developmentDependency="true" />
```
Comment (MichaelKetting) [inline Build/packages.config:12]: update in .nuget/packages.config is missing. See also https://github.com/re-motion/Relinq-SqlBackend/pull/4/files#r422187249

**Pull request merged**